mlir/utils/performance/perfRunner.py

```diff
@@ -766,7 +814,7 @@ def foundExternalTool(paths: Paths, opType: Operation):
     if opType == Operation.CONV and not paths.miopen_driver_path:
         return False
     if opType == Operation.GEMM and \
-            (not paths.mlir_paths or not paths.mlir_paths.rocblas_benchmark_driver_path):
+            (not paths.mlir_paths and (not paths.mlir_paths.rocblas_benchmark_driver_path or not paths.mlir_paths.ck_benchmark_driver_path)):
```

Comment:
Maybe instead pass in `Optional[GEMMLibrary]` here to make it a more precise check.